torch/testing/_internal/common_utils.py

```diff
@@ -2499,3 +2499,10 @@ def has_breakpad() -> bool:
         return True
     except RuntimeError as e:
         return False
+
+def find_library_location(condition: bool, lib_name: str) -> Path:
```

Comment:
`condition` is not a very informative parameter name, can it be changed to something else?

Reply:
agreed. I have removed this one